test/core/application/app_config_test.cpp

```diff
@@ -43,9 +74,15 @@ TEST_F(AppConfigurationTest, DefaultValuesTest) {
       get_endpoint("0.0.0.0", 40363);
   boost::asio::ip::tcp::endpoint const ws_endpoint =
       get_endpoint("0.0.0.0", 40364);
-  char const *args[] = {"/path/"};
+  char const *args[] = {"/path/",
+                        "--genesis",
+                        "genesis_path",
+                        "--leveldb",
+                        "leveldb_path",
+                        "--keystore",
+                        "keystore path"};
```

Comment:
if I'm not mistaken, last element must be **nullptr**.

Reply:
This makes no sense, we have ```argc``` to determine number of args.